itkwidgets/viewer.py

```diff
@@ -275,6 +275,21 @@ def set_image_volume_sample_distance(self, distance: float):
     def set_image_volume_scattering_blend(self, scattering_blend: float):
         self.queue_request('setImageVolumeScatteringBlend', scattering_blend)
 
+    def compare_images(self, fixed_image: Image_Param = 'Fixed', moving_image: Image_Param = 'Image', method: str = 'checkerboard', pattern: tuple[int, int, int] = [4, 4, 4], swap_image_order: bool = False):
```

Comment:
```
from typing import Tuple
[...]

Tuple[int, int, int]
```

`Image_Param` -> `Union[str, Image]`

I think we should emit default values for fixed_image, moving_image.

Reply:
Yes, removed default values for fixed_image and moving_image.